### PR TITLE
HURUmap Tanzania update Story Showcase images and Partner Logos 

### PR DESCRIPTION
**Pull request by josephluvanda**
## Description

This change updates story showcase and images, partner logos and adds 'Number of desks' dataset(Disabled, pending data discrepancy fix)

Fixes # 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
screenshot-127 0 0 1-8000-2017-12-20-12-35-15-007

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have  performed a self-review of my own code
- [x] I have commente] my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: hurumap/static/css/hurumap.css [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+  color: #fff !important;
+  background-color: #1c9d95 !important;
+  border-color: #1c9d95 !important;
+  border-radius: 0 !important;
+  font-weight: 200 !important;
+  font-family: "Poppins", sans-serif !important;
+  padding: 3px 6px 1px !important; }
+
+.how-it-works img {
+  max-width: 70px; }
```
Comment (DavidLemayian) [inline hurumap/static/css/hurumap.css]: What's this file doing here?

File: hurumap_tz/static/css/hurumap.css [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  color: #fff !important;
+  background-color: #1c9d95 !important;
+  border-color: #1c9d95 !important;
+  border-radius: 0 !important;
+  font-weight: 200 !important;
+  font-family: "Poppins", sans-serif !important;
+  padding: 3px 6px 1px !important; }
+
+.how-it-works img {
+  max-width: 70px; }
```
Comment (DavidLemayian) [inline hurumap_tz/static/css/hurumap.css]: What's this file for?

File: hurumap_tz/static/css/hurumap.scss [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+  font-weight: 200 !important;
+  font-family: $font-poppins !important;
+  padding: 3px 6px 1px !important;
+}
+
+.how-it-works {
+  img {
+    max-width: 70px;
+  }
+}
```
Comment (DavidLemayian) [inline hurumap_tz/static/css/hurumap.scss]: What's this file for?

**Comment by josephluvanda** [comment]
@DavidLemayian, I added new showcase images and they were being pulled on one side. 
I removed  `background-position: center center;` in hurumap.scss and ran `python manage.py compilescss` to position the new showcase images. Showcase with `background-position: center center;`

screenshot-127 0 0 1-8000-2018-01-09-09-10-58-933


  @

**Comment by DavidLemayian** [comment]
@phillipahereza Could you take a look at this to align best way to achieve these customisations?

**Comment by phillipahereza** [comment]
@darkdant3 change `background-size: cover` to `background-size: contain`
screen shot 2018-01-11 at 12 35 11 pm

**Comment by phillipahereza** [comment]
@darkdant3 the easiest solution in my opinion would be to change the showcase template and use the `img` tag instead of background images since the showcase images are the same size and style.
Secondly, the `hurumap.css` file is automatically generated so be careful when adding new files for git to track. I'd suggest you add the specific files instead of the global add or add the generated files to your `gitignore`

File: hurumap_tz/templates/profile/profile_detail.html [hunk 4]
```diff
@@ -358,8 +358,12 @@ <h1>Primary School Teachers</h1>
                  {% include 'profile/_blocks/_stat_list.html' with stat=primary_school_teachers.number_of_teachers stat_type='number' %}
              </div>
             <div class="column-half" id="chart-pie-primary_school_teachers-primary_school_teachers" data-stat-type="percentage" data-chart-title="Distribution of primary school teachers based on gender(2016)" data-initial-sort="-value"></div>
-
         </section>
+        <!--<section class="clearfix stat-row">
+          <div class="column-half">
+              {% include 'profile/_blocks/_stat_list.html' with stat=primary_school_desks.number_of_desks stat_type='number' %}
+          </div>
+      </section>-->
```
Comment (DavidLemayian) [inline hurumap_tz/templates/profile/profile_detail.html:366]: Would this be needed if it's commented out?

**Comment by DavidLemayian** [comment]
Good job on the update @darkdant3. @phillipahereza please approve if ready to merge.

File: hurumap_tz/templates/profile/profile_detail.html (hunk 4 above)
Comment (josephluvanda) [reply hurumap_tz/templates/profile/profile_detail.html:366]: Yes, It's pending a data discrepancy fix and some new stats

**Review by phillipahereza — APPROVED** [review]

**Pull request merged**